### DISCUSSION TITLE
Integration of two hotkeys, 'Move list and sublists up(down)' and 'Swap line up(down)'

### DISCUSSION THREAD

**Discussion by iiz00** [vslinko/obsidian-outliner#100] [Ideas]
I want to assign both of Outliner hotkey 'Move list and sublists up' and Obsidian's basic hotkey 'Swap line up' to Ctrl+↑.
If it's not my misunderstanding, assigning both of the function to the same key combination doesn't work properly.
Is it possible to fix for both hotkeys to work properly, or 'Move list and sublists' to work as swap line outside of bullet list?

**Comment by vslinko** [comment]
Thank you for your idea!

Looks like it's not possible to combine both commands.

But it possible to add "swap line" feature into "Move list and sublists up".
I'll consider it.

**Reply by iiz00** [reply]
Wow, I'll looking forward to the feature coming.
And thank you for all your great works!

**Comment by dcoales** [comment]
I'm keen on something like this too.  I have pages with headers and under each header bullets.  I can move the bullets up and down within the header but a bullet won't go down into the next header or up into the preceding one.  For that I have to swap the lines using Ctrl-V (in my set up).  It would be great if the outliner could move bullets over headers.

Thanks very much.
